lib/OpenQA/Task/Git/Clone.pm

```diff
@@ -119,7 +119,8 @@ sub _git_clone ($job, $ctx, $path, $url) {
     }
     else {
         # updating local branch to latest remote branch version
-        _git_fetch($path, "$requested_branch:$requested_branch");
+        my $dst = 'clone_' . substr($requested_branch, 0, 7);
+        _git_fetch($path, "$requested_branch:$dst");
```

Comment:
This should do it:
```suggestion
        _git_fetch($path, $requested_branch);
```

Reply:
Why did you ignore this suggestion?

Reply:
Ok, I think your change is actually the same as my suggestion, just that you shortened it because the fetch is now the same call for both cases.